web/src/components/core/SectionSkeleton.jsx

```diff
@@ -23,19 +23,27 @@ import React from "react";
 import { Skeleton } from "@patternfly/react-core";
 import { _ } from "~/i18n";
 
-const SectionSkeleton = () => (
-  <>
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="50%"
-    />
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="25%"
-    />
-  </>
-);
+const SectionSkeleton = ({ numRows = 2 }) => {
+  const WaitingSkeleton = ({ width }) => {
+    return (
+      <Skeleton
+        screenreaderText={_("Waiting")}
+        fontSize="sm"
+        width={width}
+      />
+    );
+  };
```

Comment:
NP: move that internal component outside, please.

```suggestion

const WaitingSkeleton = ({ width }) => {
  return (
    <Skeleton screenreaderText={_("Waiting")}  fontSize="sm" width={width}  />
  );
};

const SectionSkeleton = ({ numRows = 2 }) => {
  
```

It's not important for this specific example, but I'd try to keep components definition outside whenever possible. If you wonder why, have a look at the _pitfall_ section at https://react.dev/learn/preserving-and-resetting-state#different-components-at-the-same-position-reset-state. And if you are curious, I'd encourage you to read the full article.